api/v1/checks.go

```diff
@@ -382,6 +382,32 @@ type Mongo struct {
 	MongoDBCheck `yaml:",inline" json:",inline"`
 }
 
+/*
+[include:datasources/elasticsearch_pass.yaml]
+*/
+
+type Elasticsearch struct {
+	ElasticsearchCheck `yaml:",inline" json:",inline"`
+}
+
+type ElasticsearchCheck struct {
+	Description `yaml:",inline" json:",inline"`
+	URL         string `yaml:"url" json:"url,omitempty" template:"true"`
```

Comment:
I would think this is a required field.. so for making it so in the CRD, remove the `omitempty` from the notation... also the formatting seems a bit wrong... can run `go fmt ./...`